src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/TaskControllerTest.java

```diff
@@ -193,31 +198,25 @@ public void should_return_a_200_with_search_results() {
         String ccdId2 = ccdIdGenerator.generate();
 
         given
-            .iCreateATaskWithCcdId(ccdId1)
+            .iCreateATaskWithCcdId(ccdId2)
             .and()
             .iCreateATaskWithCcdId(ccdId2);
 
-        SearchParameters searchParameters = new SearchParameters(
-            emptyList(),
-            emptyList(),
-            emptyList(),
-            emptyList(),
-            ccdId2,
-            null,
-            null,
-            null
-        );
+        SearchTaskRequest searchTaskRequest = new SearchTaskRequest(asList(
+            new SearchParameter(JURISDICTION, SearchOperator.IN, singletonList("IA"))
+        ));
+
         Response result = given()
             .contentType(MediaType.APPLICATION_JSON_VALUE)
             .headers(authorizationHeadersProvider.getLawFirmAAuthorization())
-            .body(new SearchTaskRequest(singletonList(searchParameters)))
+            .body(searchTaskRequest)
             .when()
             .post("task");
 
         result.then().assertThat()
             .statusCode(HttpStatus.OK.value())
-            .body("tasks.size()", equalTo(1))
-            .body("tasks[0].caseData.reference", equalTo(ccdId2))
+            .body("tasks.caseData.jurisdiction", everyItem(is("IA")))
+            .body("tasks.caseData.reference", contains(ccdId2, ccdId1))
         ;
     }
 
```

Comment:
Can we add a test that cover OR operand. i.e. AC 3, e.g. (Then the Search API treats each parameter as if they are joined by a boolean OR (e.g. Location="12344567" OR Location="5356363")

Reply:
Added extra tests to cover that scenario